taskKey="en.codegym.task.pro.task14.task1408"\n\nSwallowing an exception

Catch the exception in the main method so that the program compiles.
When an exception occurs, display the errorMessage string.


Требования:
1.	You need to add a try-catch block in the main method.
2.	The main method must not throw checked exceptions.
3.	The main method should call the generateLuckyNumber method.
4.	Don&#39;t change the generateLuckyNumber method.
5.	When an exception occurs in the main method, you need to display the errorMessage string on the screen.


